cubedash/summary/_stores.py

```diff
@@ -1561,6 +1563,21 @@ def get_dataset_footprint_region(self, dataset_id):
             row.region_code,
         )
 
+    def get_dataset_uris(self, dataset: Dataset):
+        """
+        Get a representation of the data URIs so that they
+        can be linked to to download them.
+        """
+        dataset_doc = serialise.from_doc(dataset.metadata_doc, skip_validation=True)
```

Comment:
This parser only reads eo3 docs, which is why the tests of older datasets are failing.

Simple option may be to check if it's eo3 with [eodatasets3.utils.is_doc_eo3(doc)](https://github.com/GeoscienceAustralia/eo-datasets/blob/8f5088ce0281b9b4b465fcbfb71fdd58d3c70a83/eodatasets3/utils.py#L144-L145) and not show the list otherwise.

Reply:
If we want to support everything, a more-thorough option could be to extract the basic [eo-to-eo3 logic](https://github.com/opendatacube/datacube-explorer/blob/ee63dcb381e623dba87c07375d5b592eea952ec2/cubedash/_stac.py#L638-L640) into a function, and run that first. (but ideally we might move it into eodatasets proper)